docs/workloads/azure/data/data_engineering/data_quality_azure.md

```diff
@@ -120,3 +120,22 @@ Here's a minimal example of a configuration file:
     ]
 }
 ```
+
+## Validation Results
+
+Results of the data quality checks are stored in the `dq_output_path` location in Delta tables. The results contain the following columns:
+
+| Column Name           | Description                                                                                      |
+|-----------------------|--------------------------------------------------------------------------------------------------|
+| data_quality_run_date | Timestamp of the data quality run                                                                |
```

Comment:
Nice, all reads well. Perhaps add a type column?
Nit: Full stops at the end of descriptions.